Example/Pellicola_Tests/DataStorageTests.swift

```diff
@@ -97,7 +99,9 @@ class DataStorageTests: XCTestCase {
         dataStorage.removeImage(withIdentifier: array[1].1)
         array.remove(at: 1)
         
-        XCTAssertEqual(array.compactMap{$0.0}, dataStorage.getImagesOrderedBySelection())
+        dataStorage.getImagesOrderedBySelection { urls in
+            XCTAssertEqual(array.count, urls.count)
```

Comment:
⚠️ This call is now asynchronous. Wait for the callback to be invoked with an expectation.